cypress-tests/cypress/support/pageBuilder/pbPublishPage.ts

```diff
@@ -0,0 +1,57 @@
+import { gqlClient } from "../utils";
+
+declare global {
+    // eslint-disable-next-line @typescript-eslint/no-namespace
+    namespace Cypress {
+        interface Chainable {
+            pbPublishPage(id: string): Chainable<Promise<any[]>>;
+        }
+    }
+}
+
+const PUBLISH_PAGE = /* GraphQL */ `
+    mutation PbPublishPage($id: ID!) {
+        pageBuilder {
+            publishPage(id: $id) {
+                data {
+                    id
```

Comment:
Let's remove redundant fields here and sync what's left with the return type in L7.